packages/ag-charts-website/e2e/examples.spec.ts

```diff
@@ -119,6 +119,11 @@ const exampleOptions: Record<string, Record<string, ExampleOverrides>> = {
         // Buttons have no visible rendering change
         'interaction-range': { skipCanvasUpdateCheck: true },
     },
+
+    sparklines: {
+        // FWs not ready yet
```

Comment:
Nit: Could we instead assert that we expect a specific failure for FWs, I think it would be quite easy to forget to re-enable these.

Reply:
https://ag-grid.atlassian.net/browse/AG-13119